.github/workflows/dotnetcore.yml

```diff
@@ -2,6 +2,10 @@ name: .NET Core CI
 
 on: [push, pull_request]
 
+  strategy:
+    matrix:
+      dotnet-version: ['2.1.803', '3.1.101']
+
 jobs:
   test:
     name: Test on ${{ matrix.os }}
```

Comment:
@karolswdev can you change the name from `Test on ${{ matrix.os }}` to something like `Test ${{ matrix.dotnet-version }} on ${{ matrix.os }}`

Reply:
Besides this little tweak it LGTM @stevespringett

Reply:
Hi @coderpatros - I just created 88ff964 that addresses this, thanks